recipes/pydsd/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PyDSD" %}
+{% set version = "1.0.3.2" %}
+{% set sha256 = "f0b815e6a1356e56a92e505cf67530cfef69588cceeebf9489f5ad5a5fc036c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytmatrix 
+    - versioneer
+  run:
+    - python
+    - pytmatrix 
+    - scipy
+    - numpy
+    - matplotlib
+    - versioneer
+    - netcdf4
+
+
+test:
+  imports:
+    - pydsd
+
+about:
+  home: http://github.com/josephhardinee/PyDSD
+  license: GPL-2.0
+  license_family: GPL
+  #license_file: LICENSE.txt
```

Comment:
The `license_file` is missing from the source distribution since it's not listed in the `MANIFEST.in`.  You should manually package the license by adding a copy in the this recipe's folder in the PR and following:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

I've added a PR to include the license in the upstream src distribution here:

https://github.com/josephhardinee/PyDSD/pull/59

Add the url as a comment in the `meta.yaml` as a reminder to use the one included with the source in the next release (assuming you merge the PR).